tests/unit/amo/components/TestTooltipMenu.js

```diff
@@ -29,21 +29,6 @@ describe(__filename, () => {
     expect(opener).toHaveClassName('MyClass');
   });
 
-  it('accepts a custom className', () => {
```

Comment:
No changes were made to `TooltipMenu`, so there shouldn't be any changes to this test.